Expensify/App
# Open the last viewed chat after signing off or going to the root page

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/141575

### Tests
1. Go to a chat and refresh the page, make sure you're still on the chat
1. Modify the URL to go to the root `/` and verify you are taken back to the last chat you were looking at
1. Sign out, and immediately sign back in and verify you are taken back to the last chat you were looking at
1. Sign out, modify the URL to go to the root `/`. 
1. Sign in, and verify you are taken back to the last chat you were looking at.
